spec.html

```diff
@@ -4682,13 +4699,165 @@ <h1>The ClassFieldDefinition Record Specification Type</h1>
           </tr>
           <tr>
             <td>
-              [[Initializer]]
+              [[Kind]]
```

Comment:
Since the table is heavily expanded from two items to eight, here is a chance to order it alphabetically as well.